UKHomeOffice/sgmr-service
# SPR-120 Remove import aliases in prep for package updates

## Description
Newer versions of Jest and Webpack don't support the alias mapping we use, this PR changes our imports to relative paths

## To test

- run app locally 
- >app should build and run
- click through pages/forms etc. 
- > everything should work as normal
- run `npm run lint -- .`
- > there should be no errors
- run `npm test`
- > there should be no errors

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
